src/e2e/resources/data/AutomatedSessionRemindersE2ETest_SqlEntities.json

```
@@ -0,0 +1,10 @@
+{
+  "accounts": {
+    "instructorWithEvals": {
+      "googleId": "tm.e2e.AutSesRem.instructor",
+      "name": "Test Ins for Aut Sessions Reminder",
+      "email": "AutSesRem.instructor@gmail.tmt",
+      "readNotifications": {}
+    }
```

Need to add id and remove readNotifications (according to Account entity):
```suggestion
    "instructorWithEvals": {
      "id": "00000000-0000-4000-8000-000000000001",
      "googleId": "tm.e2e.AutSesRem.instructor",
      "name": "Test Ins for Aut Sessions Reminder",
      "email": "AutSesRem.instructor@gmail.tmt",
    }
```